js/main.js

```diff
@@ -1,10 +1,11 @@
 import {generateAds} from './render.js';
 import {offersArray} from './offers-array.js';
-import {togglePageActiveState} from './form.js';
+import {togglePageActiveState, validateRoomsInput, roomsQty} from './form.js';
 
 const similarOffer = document.querySelector('#map-canvas');
 
 generateAds(offersArray);
 similarOffer.appendChild(generateAds(offersArray));
 
-togglePageActiveState(true);
+togglePageActiveState(false);
+roomsQty.addEventListener('change', (validateRoomsInput));
```

Comment:
за все, связанное с формой отвечает модуль **form**, давай навешивание обработчика там и оставим, другие обработчики же ты в **form** навешиваешь, чем этот отличается?

Reply:
также обертка вокруг колбека и доп переменная **roomsQty** не нужны:
```suggestion
searchForm.rooms.addEventListener('change', validateRoomsInput);
```